docs/_docs/deployment-methods.md

```diff
@@ -203,6 +203,6 @@ Setting up Kickster is very easy, just install the gem and you are good to go. M
 
 ## Aerobatic
 
-[Aerobatic](https://www.aerobatic.com) is an add-on for Bitbucket that brings GitHub Pages style functionality to Bitbucket users. It includes continuous deployment, custom domains with a wildcard SSL cert, CDN, basic auth, and staging branches all in the box.
+[Aerobatic](https://www.aerobatic.com) has custom domains, global CDN distribution, basic auth, CORS proxying, and a growling list of plugins all included.
```

Comment:
sorry! Was working from a really bad airport wifi and SSH connection. Will fix right away!